src/fancy-tools/scaleOverlay.js

```diff
@@ -0,0 +1,242 @@
+import external from './../externalModules.js';
+import baseTool from './../base/baseTool.js';
+// Drawing
+import { getNewContext, draw, path } from '../util/drawing.js';
+
+const cornerstone = external.cornerstone;
+
+const drawLine = (context, startPoint, endPoint) => {
```

Comment:
Can we use the `drawLine` from `util/drawing.js` instead

Reply:
yeah, I finished this first part, with the idea of just make it work. 
I will refactor :)